mlreco/models/layers/common/gnn_full_chain.py

```diff
@@ -435,8 +436,8 @@ def run_particle_gnns(self, result, input, frag_result):
                            'node_pred_type': 'node_pred_type',
                            'node_pred_p': 'node_pred_p',
                            'node_pred_vtx': 'node_pred_vtx',
-                           'input_node_features': 'input_node_features',
-                           'input_edge_features': 'input_edge_features'}
+                           'input_node_features': 'particle_node_features',
+                           'input_edge_features': 'particle_edge_features'}
```

Comment:
Why this renaming? Can you remind me what module(s) consume `input_node_features` and `input_edge_features` ?

Reply:
I've placed `input_xxxx_features` before for debugging purposes. I think we don't use input node features anywher except for vertex-net (which is under-construction anyway)

The name change was required because I needed to separate geometric encoder's output for 1) shower fragments 2) track fragments 3) particles (superset of fragments).

Reply:
Okay that sounds good, thanks for clarifying